app/manual/shared.html

```diff
@@ -1,35 +1,480 @@
+    A step during matching elicitation, asking the user to match two different
+    scenarios.
+  </div>
+</div>
+
+<h4 id="mcda-precise-swing-elicitation">Precise swing weighting</h4>
+
+<p>
+  Precise swing weighting elicitation is an abstract method of preference
+  elicitation that lets the user directly indicate how important they find each
+  criterion's improvement to be. First, the user is asked which criterion is the
+  most important to them. Then, they are presented with a table containing all
+  criteria, including their worst and best values (given the defined scale
+  ranges) and a slider that lets them indicate the importance of that criterion
+  relative to the most important one. These importances are normalised to sum up
+  to 1 to calculate the weights used in analysis. Note that are derived from
+  these importances,
+</p>
+<div class="figure">
+  <img src="img/preciseSwingElicitation.png" />
+  <div class="caption">
+    The second step of precise swing weighting elicitation, asking the user to
+    indicate the importance of each criterion relative to the most important
+    one.
+  </div>
+</div>
+
+<h4 id="mcda-imprecise-swing-elicitation">Imprecise swing weighting</h4>
+
+<p>
+  Imprecise swing weighting elicitation works like
+  <a href="#mcda-precise-swing-elicitation">Precise swing weighting</a>, with
+  one difference: in the second step, the user can specify the improtance of
+  each secondary criterion using a two-valued slider, instead of having to pick
+  a single value. These two values indicate the minimum and maximum weight a
+  user is willing to assign to the criterion. This means that the weights are
+  uncertain, which allows <a href="#mcda-smaa">SMAA</a> to operate on the
+  preferences, besides any uncertainty that might be present in the data. Note
+  that the uncertainty of imprecise swing is treated as a uniform distribution,
+  meaning that each point in the specified range is treated as equally likely.
+</p>
+<div class="figure">
+  <img src="img/impreciseSwingElicitation.png" />
+  <div class="caption">
+    The second step of imprecise swing weighting elicitation, asking the user to
+    indicate the range of the importance of each criterion relative to the most
+    important one.
+  </div>
+</div>
+
+<h4 id="mcda-tradeoff-table">Trade off table</h4>
+
+<p>
+  For many users, it is helpful to illustrate the concrete consequences of the
+  abstract weights which result from preference elicitation. The trade off table
+  below the weights table does so by showing how much each secondary criterion
+  should change to be compensate for a certain change in a reference criterion.
+  Any criterion can be chosen as the reference, and the values between which
+  this reference criterion changes can also be set by the user.
+</p>
+<div class="figure">
+  <img src="img/tradeOffTable.png" />
+  <div class="caption">
+    The trade off table, showing how much each secondary criterion would have to
```

Comment:
Maybe use an example picture in which one of the values goes into the absurd (and thus showing the warning)? I don't think it's really necessary, but might be nice.